packages/SwingSet/misc-tools/store-stats/src/store-stats.js

```diff
@@ -0,0 +1,160 @@
+import { StatsD } from 'hot-shots';
```

Comment:
looks like this needs to be added to the `package.json`, and the `yarn.lock` updated

Reply:
Oh, it's already there. I bet the lint CI job doesn't believe in packages that aren't direct subdirectories of the top-level `packages/` workspace.

Let's just disable lint on this line, I think by inserting
```
// eslint-disable-next-line
```
before it.. see if that appeases CI.